rm-config/src/main_config.rs

```diff
@@ -50,19 +52,79 @@ fn default_refresh() -> u64 {
     5
 }
 
+#[derive(Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
+pub enum Header {
+    Name,
+    SizeWhenDone,
+    Progress,
+    Eta,
+    DownloadRate,
+    UploadRate,
+    DownloadDir,
+}
+
+impl Header {
+    pub fn default_constraint(&self) -> Constraint {
+        match self {
+            Header::Name => Constraint::Max(70),
+            Header::SizeWhenDone => Constraint::Length(12),
+            Header::Progress => Constraint::Length(12),
+            Header::Eta => Constraint::Length(12),
+            Header::DownloadRate => Constraint::Length(12),
+            Header::UploadRate => Constraint::Length(12),
+            Header::DownloadDir => Constraint::Max(70),
```

Comment:
since the padding header cell will be removed, could we add some extra padding within the `name` tab? or maybe another way to add extra spacing between the `name` and `size` column?

Reply:
above can be ignored if padding header is planned to be added